### ISSUE TITLE
Error when trying to choose the location of the markets

### ISSUE THREAD

**Issue by robson-alcantara** [NearbyShops/Nearby-Shops-Android-app#33]
When trying to choose a market location, the map is not displayed. The following message is displayed on the console: "Unexpected response code 400 for https://clients4.google.com/glm/mmap/api"

Is it due to some key? Is it possible to access the map from the compilation of the source code?

**Comment by SumeetMoray** [comment]
yes i deactivated my key. because you are charged for google maps. 
You need to generate your own api key and use it !

**Comment by robson-alcantara** [comment]
Right. Thanks.

**Comment by robson-alcantara** [comment]
Did this disabled key affect the Play Store version? If so, is it still possible to use the application?
